sdk/src/main/java/com/mapbox/maps/renderer/widget/BitmapWidgetRenderer.kt

```diff
@@ -80,16 +79,16 @@ internal class BitmapWidgetRenderer(
     )
   }
 
-  private fun topY() = when (position.vertical) {
-    WidgetPosition.Vertical.TOP -> marginY + halfBitmapHeight
-    WidgetPosition.Vertical.CENTER -> surfaceHeight.toFloat() / 2 + marginY
-    WidgetPosition.Vertical.BOTTOM -> surfaceHeight.toFloat() - (halfBitmapHeight + marginY)
+  private fun topY() = when (widgetPosition.vertical) {
+    WidgetPosition.Vertical.TOP -> widgetPosition.offsetY + halfBitmapHeight
+    WidgetPosition.Vertical.CENTER -> surfaceHeight.toFloat() / 2 + widgetPosition.offsetY
+    WidgetPosition.Vertical.BOTTOM -> surfaceHeight.toFloat() - (halfBitmapHeight + widgetPosition.offsetY)
```

Comment:
If I understand right, this is a bit inconsistent. If `TOP` and `CENTER` then a positive `offsetY` pushes it down, if `BOTTOM` a positive `offsetY` pushes it up.
Will users understand that `offsetY` needs to be positive to "move" things up or should folks pass a negative `offsetY` to shift it up?

(I guess similar with the `leftX`)

Reply:
@jush I didn't update the logic here, it behaves more like a padding, should we name it as padding instead? Or do you think we should just make it offset and let user decide if they need to offset to a negative number?

Reply:
To be consistent with the offset always having the same origin (top-left), I would let the user decide if they need to use a negative number (e.g. in the `BOTTOM` and `RIGHT` cases)

Reply:
@yunikkk what's your thoughts here? include a padding or use an absolute offset towards the same origin?

For Android Auto compatibility issues, we might want to provide the original API as deprecated methods.

Reply:
I've treated them differently since they were `margin`s previously, now I would agree with making offsets behave like Android `translation` - e.g. positive values move widget to bottom/right corner and negative to the top-left.

Reply:
Btw what are `android auto compatibility issues` ? Do you mean compatibility with Nav SDK?

Reply:
Please see the compatibility table in https://github.com/mapbox/mapbox-maps-android/tree/main/extension-androidauto#compatibility-with-maps-sdk-v10

The compatibility means both compatibility between Android Auto extension and Maps SDK(where Android Auto extension extends the BitmapWidget), and between Maps SDK and Nav SDK(where nav SDK uses setTranslation API in their Android Auto drop-in UI [ref1](https://github.com/mapbox/mapbox-navigation-android/blob/5dc78ef37babc548d63ad6725509bd5723bc2515/libnavui-androidauto/src/main/java/com/mapbox/androidauto/navigation/speedlimit/CarSpeedLimitRenderer.kt#L122) [ref2](https://github.com/mapbox/mapbox-navigation-android/blob/5dc78ef37babc548d63ad6725509bd5723bc2515/libnavui-androidauto/src/main/java/com/mapbox/androidauto/map/logo/CarLogoSurfaceRenderer.kt#L32).)

If we introduce breaking changes to these libraries, it means the Nav SDK and Android Auto extension will not be compatible with newer version of Maps SDKs.

Reply:
Settled with using absolute offset towards the same direction.